src/Sylius/Bundle/WebBundle/Resources/config/routing/backend/content/block/action_block.yml

```diff
@@ -26,8 +26,9 @@ sylius_backend_action_block_update:
         _controller: sylius.controller.action_block:updateAction
         _sylius:
             template: SyliusWebBundle:Backend/Content/Block/Action:update.html.twig
-            method: find
-            arguments: [$id]
+            repository:
+                method: find
+                arguments: [$id]
```

Comment:
IMHO it would be better DX if such settings would be default ones, so you don't need to set them and just set:
```yaml
repository: ~